loopchain/consensus/syncer.py

```diff
@@ -0,0 +1,148 @@
+import asyncio
+import time
+from typing import TYPE_CHECKING, Dict
+
+from lft.consensus.events import ReceiveDataEvent, ReceiveVoteEvent
+
+from loopchain import configure as conf
+from loopchain.blockchain.blocks.v1_0 import Block
+from loopchain.blockchain.votes.v1_0 import BlockVote
+from loopchain.channel.channel_property import ChannelProperty
+from loopchain.protos import loopchain_pb2, loopchain_pb2_grpc
+from loopchain.tools.grpc_helper import GRPCHelper
+
+if TYPE_CHECKING:
+    from loopchain.peer.block_manager import BlockManager
+    from lft.event import EventSystem
+
+
+class Syncer:
+    def __init__(self,
+                 block_manager: 'BlockManager',
+                 event_system: 'EventSystem',
+                 last_block_height: int = 0):
+        self._block_manager = block_manager
```

Comment:
How about adding an attribute `self._blockchain = self._block_manager.blockchain` for all scope in this class. I think `blockchain` is using frequently in this file. I suppose to be better readability of this class if you accept it.